src/System.Private.CoreLib/shared/System/Runtime/InteropServices/Marshal.NoCom.cs

```diff
@@ -37,20 +38,22 @@ public static void CleanupUnusedObjectsInCurrentContext()
 
         public static IntPtr CreateAggregatedObject<T>(IntPtr pOuter, T o)
         {
+            // TODO-NULLABLE: Generic T cannot be null
             throw new PlatformNotSupportedException(SR.PlatformNotSupported_ComInterop);
         }
 
-        public static object CreateWrapperOfType(object o, Type t)
+        public static object? CreateWrapperOfType(object? o, Type t)
         {
             throw new PlatformNotSupportedException(SR.PlatformNotSupported_ComInterop);
         }
 
         public static TWrapper CreateWrapperOfType<T, TWrapper>(T o)
         {
+            // TODO-NULLABLE: Generic T can be null
```

Comment:
I think this can be represented with `where T: class?`

Reply:
That looks to add a `where T : class` constraint, which changes the signature

Reply:
Yes, we would need `[MaybeNull]` or `where T : object?`.

We've been tagging these with `// TODO-NULLABLE-GENERIC

Reply:
@stephentoub would you prefer I change it to: `// TODO-NULLABLE-GENERIC: T can be null`? I think the additional note on whether T can or cannot be null is useful

Reply:
No preference... but pretty much all of the cases where we have TODO-NULLABLE-GENERIC are in the same camp.

Reply:
Fixed.